.gitignore

```diff
@@ -132,6 +132,4 @@ dmypy.json
 # Models
 *.onnx
 # include small test model for tests
-!tests/assets/onnx/model.onnx
-
-.vscode
```

Comment:
It seems that you have removed this line, could you make sure this file is the same as https://github.com/huggingface/optimum/blob/main/.gitignore ?